ext/node/polyfills/_util/_util_callbackify.ts

```diff
@@ -27,7 +27,6 @@ import { primordials } from "ext:core/mod.js";
 const {
   ArrayPrototypePop,
   Error,
-  FunctionPrototypeApply,
```

Comment:
Can we add this primordial instead? I'm not sure how it works.

Reply:
Used `ReflectApply` instead